next/lib/graphql/fragments/nodes.ts

```diff
@@ -41,6 +41,9 @@ export const FRAGMENT_NODE_ARTICLE = graphql(`
         ...FragmentUser
       }
     }
+    translations {
+      ...FragmentNodeTranslation
+    }
```

Comment:
We have to add this fragment to each node type individually, since the property is not on the NodeInterface (makes sense, only some node types could be set as translatable).